lib/pages/feed/widgets/feed_filter_popup.dart

```diff
@@ -8,6 +8,7 @@ import 'package:campus_app/utils/widgets/campus_filter_selection.dart';
 import 'package:campus_app/utils/widgets/popup_sheet.dart';
 
 /// This widget displays the filter options that are available for the
+// ignore: comment_references
```

Comment:
Dart docs will be improved within the near future to improve the overall documentation quality.